frontend/src/app/utils/testResultCSV.ts

```diff
@@ -23,24 +23,20 @@ export function parseDataForCSV(data: any, multiplexEnabled: boolean) {
       ),
       "Patient date of birth": moment(r.patient.birthDate).format("MM/DD/YYYY"),
       "Test date": moment(r.dateTested).format("MM/DD/YYYY h:mma"),
-      ...(multiplexEnabled
-        ? {
-            "COVID-19 result":
-              TEST_RESULT_DESCRIPTIONS[
-                getResultByDiseaseName(r.results, "COVID-19") as Results
-              ],
-            "Flu A result":
-              TEST_RESULT_DESCRIPTIONS[
-                getResultByDiseaseName(r.results, "Flu A") as Results
-              ],
-            "Flu B result":
-              TEST_RESULT_DESCRIPTIONS[
-                getResultByDiseaseName(r.results, "Flu B") as Results
-              ],
-          }
-        : {
-            "COVID-19 result": TEST_RESULT_DESCRIPTIONS[r.result as Results],
```

Comment:
⚠️ There's another use of the `result` column being used on the front end that I missed 😬 

But I think this might only affect envs that don't have the feature flag enabled which is demo and training?

Reply:
If you all think it's best, I can open a separate PR for this change

Reply:
I tested the csv download in dev and it looks good. If it's just training and demo that don't have the feature flag I don't feel strongly about moving this to a new PR.

Reply:
And actually after a second look through the logic, I think it should still be fine in demo and training unless I'm misunderstanding the feature flag.